openai/src/commonMain/kotlin/com/xebia/functional/xef/auto/llm/openai/OpenAIScopeExtensions.kt

```diff
@@ -9,6 +9,13 @@ import com.xebia.functional.xef.llm.models.functions.CFunction
 import com.xebia.functional.xef.prompt.Prompt
 import kotlinx.serialization.serializer
 
+@AiDsl
+suspend fun CoreAIScope.promptMessage(
+  prompt: String,
+  model: Chat = OpenAI.DEFAULT_CHAT,
+  promptConfiguration: PromptConfiguration = PromptConfiguration.DEFAULTS,
+): String = model.promptMessage(prompt, context, promptConfiguration)
+
```

Comment:
Question! Why don't you better include this into the `CoreAIScope` as an Instance Method rather than an Extension Method?

Reply:
This is already there as well, and here it appears as a convenience to the user where it does not need to specify models and uses the defaults:
```kotlin
package com.xebia.functional.xef.auto

import com.xebia.functional.xef.auto.llm.openai.getOrElse
import com.xebia.functional.xef.auto.llm.openai.promptMessage

suspend fun main() {
  ai {
    val love: String = promptMessage("tell me you like me with just emojis")
    println(love)
  }.getOrElse { println(it) }
}
```
If you don't use the open ai imports then you have to do:
```kotlin
package com.xebia.functional.xef.auto

import com.xebia.functional.xef.auto.llm.openai.OpenAI
import com.xebia.functional.xef.auto.llm.openai.getOrElse

suspend fun main() {
  ai {
    val love: String = OpenAI.DEFAULT_CHAT.promptMessage("tell me you like me with just emojis")
    println(love)
  }.getOrElse { println(it) }
}
```